scripts/efs-cron-pod.yml

```
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: inspect-pvc-pod
+spec:
+  containers:
+  - name: inspect-pvc-container
+    image: busybox
+    command:
+      - /bin/sh
+      - -c
+      - |
+        #!/bin/sh
+        set -eux
+
+        # EFS_PERSIST_MNT is the mount point for the PVC
+        EFS_PERSIST_MNT="/mnt/efs-persist"
+        EFS_HOME="$EFS_PERSIST_MNT/home"
+
+        # Generate the size report
+        du -sh $EFS_HOME/* | awk '{print $1, $2}' > $EFS_HOME/size_report.txt
```

if you do this way (storing into EFS itself), better to IMHO
```suggestion
        cd "$EFS_HOME" && d8 -sh * | awk '{print $1, $2}' > size_report.txt
```

then you would avoid having some irrelevant path prefix there.